CMake/ClangCompilers.cmake

```diff
@@ -5,7 +5,7 @@ IF ( CMAKE_CXX_COMPILER_VERSION VERSION_LESS 3.0 )
 ENDIF()
 
 # Enable OpenMP
-SET(ENABLE_OPENMP 0)
+SET(ENABLE_OPENMP 1)
```

Comment:
This might cause problems on Mac OS, since the clang there doesn't have OpenMP (the flag is silently ignored).   It could be fixed later.